Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -178,10 +257,21 @@ - (void)tokenizePayPalAccountWithPayPalRequest:(BTPayPalRequest *)request comple
 
             [self sendAnalyticsEventForInitiatingOneTouchForPaymentType:request.paymentType withSuccess:analyticsSuccess];
 
-            [self handlePayPalRequestWithURL:approvalUrl
-                                       error:error
-                                 paymentType:request.paymentType
-                                  completion:completionBlock];
+            BOOL isPayPalCheckoutRequest = [self.payPalRequest isKindOfClass:BTPayPalCheckoutRequest.class];
```

Comment:
If we have access to the `request` in this method, for consistency, we can stick to accessing the `request` directly
```suggestion
            BOOL isPayPalCheckoutRequest = [request isKindOfClass:BTPayPalCheckoutRequest.class];
```